### ISSUE TITLE
Scaling of space-time domain

### ISSUE THREAD

**Issue by ShupengGao** [lululxvi/deepxde#1194]
Can I use the following code to scale the space-time domain:
def feature_transform(x):
        return torch.cat(
            [
                x[:,0:1],
                x[:,1:2] * 20,
                x[:,2:3] * 100,
            ],
            dim=1,
        )
net.apply_feature_transform(feature_transform)

**Comment by lululxvi** [comment]
Yes.
